app/survey_config/social_survey_config.py

```diff
@@ -11,14 +11,20 @@
 class SocialSurveyConfig(
     SurveyConfig,
 ):
-    base_url: str = "https://rh.ons.gov.uk"
     survey_title: str = "ONS Social Surveys"
     footer_links: Iterable[MutableMapping] = field(default_factory=list)
     footer_legal_links: Iterable[Mapping] = field(default_factory=list)
 
     def __post_init__(self):
         super().__post_init__()
 
+        self.base_url = "https://rh.ons.gov.uk"
+        self.contact_us_url = f"{self.base_url}/contact-us/"
+        self.cookie_settings_url = f"{self.base_url}/cookies/"
+        self.privacy_and_data_protection_url = (
+            f"{self.base_url}/privacy-and-data-protection/"
+        )
```

Comment:
- This is already called in the `__post_init__` should be removed. 
- The base URL shouldn't be hardcoded here as it would override the account service URL set by the env var/cookie. When creating the social config for the non cookie pages, you will need to pass it the base url to use, otherwise it will get it from the cookie.

You will need to pass in a base url when you want the social survey config without the

Reply:
This one is slightly confusing because of all the relations between the classes and conditions we need to meet, I left the hardcoded `base_url` but not in `post init`, only top level of social config class for the sake of correct `account_service_log_out_url` being generated for social. Otherwise `base_url` is initialised differently in template helpers module now:
```python
base_url = "https://rh.ons.gov.uk" if survey_theme == SurveyType.SOCIAL else (
        cookie_session.get("account_service_base_url") or ACCOUNT_SERVICE_BASE_URL
    )
```
that seems to do the work for us?

Reply:
if we don't want hardcoded rh link in class init should we just?:
```python
if not self.account_service_log_out_url:
            self.account_service_log_out_url: str = "https://rh.ons.gov.uk/sign-in/logout"
```